source/opcode.cpp

```diff
@@ -731,3 +731,16 @@ bool spvOpcodeIsAccessChain(SpvOp opcode) {
       return false;
   }
 }
+
+bool spvOpcodeIsBit(SpvOp opcode) {
+  switch (opcode) {
+    // TODO(https://github.com/KhronosGroup/SPIRV-Tools/issues/3557):
+    // Right now, only certain operations are supported.
+    case SpvOpBitwiseOr:
+    case SpvOpBitwiseXor:
+    case SpvOpBitwiseAnd:
+      return true;
+    default:
+      return false;
+  }
```

Comment:
I don't feel comfortable merging this in a TODO state, as there aren't that many bitwise operations and someone might decide to use this function without looking at the implementation (assuming it works for all of them).

Would you be happy to go ahead and get this working for all opcodes?

An alternative is to have this function handle all opcodes, but then in spirv-fuzz have a TODO to explain that the transformation doesn't yet support all of them.

Reply:
Done.